integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateListeners.scala

```diff
@@ -569,9 +577,8 @@ object CommitPreAggregateListener extends OperationEventListener with CommitHelp
       if (commitFailed) {
         sys.error("Failed to update table status for pre-aggregate table")
       }
-    }
-
-
+       operationContext.setProperty("commitComplete", !commitFailed)
```

Comment:
In case of any exception this will not get set and the load would be marked as a success.